worker/api/api.proto

```diff
@@ -11,6 +11,28 @@ service ToWorker {
   rpc RequestStreamEnd (EndStreamRequest) returns (Status) {}
   rpc RequestWaveform (WaveformRequest) returns (WaveFormResponse) {}
   rpc RequestCut (CutRequest) returns (CutResponse) {}
+  rpc CleanSectionImageFolder(CleanSectionImageFolderRequest) returns (Status) {}
+  rpc GenerateSectionImage (GenerateSectionImageRequest) returns (GenerateSectionImageResponse) {}
+}
+
+message CleanSectionImageFolderRequest {
+  string CourseName = 1;
+  uint32 CourseYear = 2;
+  string CourseTeachingTerm = 3;
+}
+
+message GenerateSectionImageResponse {
+  string Path = 1;
+}
+
+message GenerateSectionImageRequest {
+  string PlaylistURL = 1;
+  string CourseName = 2;
+  uint32 CourseYear = 3;
+  string CourseTeachingTerm = 4;
+  uint32 Hours = 5;
+  uint32 Minutes = 6;
+  uint32 Seconds = 7;
 }
```

Comment:
If we make this request send all sections that should be generated, we can omit `CleanSectionImageFolder` and do the cleanup in this request.

Reply:
Put all sections into one request. Works great, thanks!